### DISCUSSION TITLE
Tecnologias do  projeto

### DISCUSSION THREAD

**Discussion by gacensadev** [garodriguesfs10/VirtualShop#2] [Q&A] [answered]
Tenho uma duvida sobre as tecnologias utilizadas no projeto

**Comment by garodriguesfs10** [comment, marked as answer]
São elas:
MicroServiços
Asp.Net MVC (Razor View)
.NET Core 6
AutoMapper
OAuth 2.0
IdentityServer
EF Core
Features:
